pyaedt/emit_core/results/revision.py

```diff
@@ -159,6 +159,11 @@ def run(self, domain):
                     raise ValueError("The domain must not have channels specified.")
         self._load_revision()
         engine = self.emit_project._emit_api.get_engine()
+        if self.emit_project._aedt_version < "2024.1":
+            if len(domain.interferer_names) == 1:
+                engine.max_simultaneous_interferers = 1
+            if len(domain.interferer_names) > 1:
+                raise ValueError("Multiple interferers cannot be specified prior to AEDT version 2023 R2.")
```

Comment:
@myoung301 "prior to 2032 R2"?  I'm guessing you meant 2024.1.  However, even that isn't accurate as we deferred the capability of specifying multiple specific tx's, so it is likely to be 2024.2 before we allow this.  But then again, none of that really matters since it isn't currently possible to hit that line (you can't specify multiple interferers in pyaedt and if you take a worst case domain that already has them specified it would run into the channels specified error on line 159).

All of which is to say, maybe you should change this, but only so as not to confuse the next person that works on this code.